tests/test_examples.py

```diff
@@ -37,23 +37,17 @@ def test_kazeeki(self):
             'description': (
                 'We are looking for freelancers ( individuals and companies ) '
                 'who offer their services related to Architecture Walkthrough '
-                'and 3D animations. Please consider this job as a potential '
-                'to several more and a long term relationship.   We are a '
-                'Media'),
+                'and 3D animations. Please consider this job as a potential to '
```

Comment:
I honestly wasn't positive what this test was doing, so I changed the example data to match what it should be.

Reply:
We can discuss today.

Reply:
I will take a look at this for an hour to see where the data was going.